packages/client/hmi-client/src/utils/math.ts

```diff
@@ -229,6 +229,7 @@ export const cleanLatexEquations = (equations: Array<string>): Array<string> =>
 				.replaceAll('\\left', '')
 				.replaceAll('{align}', '')
 				.replaceAll('=&', '=')
+				.replaceAll('*', ' *') // this fixes some parsing issues with variables such as '\beta' and '\gamma' with a '*' right after
```

Comment:
Please be more specific with comments and intents: `this fixes some parsing issues...` is rather vague

Reply:
i made the comment clearer hopefully, essentially scientific variable such as \beta and \gamma would not be parsed if there was just '*' character following them, but it was fine with other math operators.